### PR TITLE
Taphunter parser

### PR DESCRIPTION
**Pull request by mjcarroll**
Addresses #20

### CODE REVIEW

**Comment by drewbrew** [comment]
I did some massive rebasing which caused quite a few conflicts here. Sorry.

**Comment by drewbrew** [comment]
Looks good. Django-fy it and add tests as in #39 and I'll merge.

File: tap_list_providers/parsers/taphunter.py [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+        beer = {
+            'name': tap['beer']['beer_name'],
+            'style': {
+                'name': tap['beer']['style'],
+                'category': tap['beer']['style_category'],
+            }
+        }
+
+        if tap['beer']['abv']:
+            beer['abv'] = float(tap['beer']['abv'])
```
Comment (drewbrew) [inline tap_list_providers/parsers/taphunter.py]: might as well go straight to `decimal.Decimal` here

**Comment by mjcarroll** [comment]
Closing this one, as it all lands in the other PR.

**Pull request closed without merging**